examples/http3_client.py

```diff
@@ -396,6 +398,13 @@ def save_session_ticket(ticket: SessionTicket) -> None:
     parser.add_argument(
         "-v", "--verbose", action="store_true", help="increase logging verbosity"
     )
+    parser.add_argument(
+        "-lp",
```

Comment:
I'm not sure the short version gains us anything, maybe drop it?

Reply:
I've dropped it. Personally I liked it, but it's not a must-have, and can be added later if needed.